tests/pywrap-opt-sowreap/input.dat

```diff
@@ -11,4 +11,4 @@ set {
    geom_maxiter 6
 }
 
-optimize('mp3',mode='soW',dertype='energy')
+optimize('mp3',mode='sow',dertype='energy')
```

Comment:
Not sure why that was lowered. Kwarg parsing works fine. 

-Daniel Smith
Sent from my iPhone. 

> On May 4, 2016, at 23:27, Lori A. Burns <notifications@github.com> wrote:
> 
> In tests/pywrap-opt-sowreap/input.dat:
> 
> > @@ -11,4 +11,4 @@ set {
> >     geom_maxiter 6
> >  }
> >  
> > -optimize('mp3',mode='soW',dertype='energy')
> > +optimize('mp3',mode='sow',dertype='energy')
> Did this not work with the awkward capitalization? If not, then i'd rather fix the kwargs case handling.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>